qctrlopencontrols/dynamic_decoupling_sequences/driven_controls.py

```diff
@@ -119,6 +119,7 @@ def convert_dds_to_driven_control(
     minimum_segment_duration : float, optional
         If set, further restricts the duration of every segment of the Driven Controls.
         Defaults to 0, in which case it does not affect the duration of the pulses.
+        Must be greater or equal to 0, if set.
```

Comment:
Good point! I think this is a reasonably requirement.

Reply:
Yeah, I was getting away without testing it explicitly before, but if I eliminate the gap > 0 test then I have to make sure this isn't negative